pkgs/development/libraries/liblqr-1/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib, stdenv, fetchurl, pkg-config, glib }:
 
 stdenv.mkDerivation rec {
-  name = "liblqr-1-0.4.2";
+  pname = "liblqr-1";
+  version = "0.4.2";
 
   src = fetchurl {
-    url = "${meta.homepage}/local--files/en:download-page/${name}.tar.bz2";
+    url = "http://liblqr.wikidot.com/local--files/en:download-page/${pname}-${version}.tar.bz2";
```

Comment:
```suggestion
    url = "https://liblqr.wdfiles.com/local--files/en%3Adownload-page/${pname}-${version}.tar.bz2";
```

Reply:
maybe we should change it to github? https://github.com/carlobaldassi/liblqr

Reply:
They announced that here http://liblqr.wikidot.com/forum/t-905211/code-hosting-moved-to-github which is linked from http://liblqr.wikidot.com/ so we can do that.

I just pasted the link I got when I visited the download page but you are right. Not sure what happened here.